lib/csv_export.rb

```diff
@@ -0,0 +1,70 @@
+module CsvExport
+  require 'csv'
+  require 'zip'
+
+  # generates a csv from the given model data
+  def generate_csv(data)
+    objects = data.first
+    columns = data.size == 1 ? objects.first.attributes.keys : data[1]
```

Comment:
I'm pretty sure I did this so that `generate_csv` and `generate_zip` would take the same kind of input. `data` including both is relevant to `generate_zip` because it couples each set with its columns. Do we value consistency more than this readability? I'm thinking I should just rename the parameter `data` to `table`